src/pages/Admin/AdminVenueRoomDetails.tsx

```diff
@@ -115,19 +117,15 @@ export const AdminVenueRoomDetails = ({
                   }
                 </div>
                 <div className="toggle-room">
-                  <label id={"toggle-" + index} className="switch">
-                    <input
-                      type="checkbox"
-                      id={"toggle-" + index}
-                      name={"toggle-" + index}
-                      checked={room.isEnabled}
-                      onClick={() => {
-                        updateRoom(!room.isEnabled);
-                      }}
-                    />
-                    <span className="slider round"></span>
-                  </label>
-                  <div>Turn room {room.isEnabled ? "Off" : "On"}</div>
+                  <Toggler
+                    id={"toggle-" + index}
```

Comment:
```suggestion
```

Reply:
removed `id` in a64982d7f2094afc0571f671312f576d4007dbcb